new_client/src/auth/Signup.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Container, Paper, Grid, TextField, Button } from '@material-ui/core';
+import "./Error.css";
+
+export default function Signup() {
+	const [form, updateForm] = React.useState({
+		username: "",
+		email: "",
+		password: "",
+		graduationYear: null,
+	});
+
```

Comment:
The schema for user requires more fields

Reply:
Resolved as @AliabbasMerchant mentioned.